hydrate/scrape.py

```diff
@@ -0,0 +1,54 @@
+"""Scrapes Github for Fabrikate Component Information."""
+from requests import get
+
+from .component import Component
+
+# URL to the Fabrikate Component Definitions
+COMP_DEFS_URL = "https://api.github.com/repos/microsoft/fabrikate-definitions/contents/definitions"
+
+
+def get_repo_components():
+    """Return the Fabrikate Component List."""
+    json_string = json_get(COMP_DEFS_URL)
+    components = parse_json(json_string)
+    components = remove_fabrikate_prefix(components)
+    for component in components:
+        print(component)
+    return components
+
+
+def parse_json(json_list):
+    """Parse json to get each component."""
+    components = []
+    for entry in json_list:
+        component = Component(entry["name"], source=entry["html_url"])
+        components.append(component)
+    return components
+
+
+def remove_fabrikate_prefix(components):
+    """Remove the fabrikate prefix from the Component names."""
+    fabrikate_found = False
+    for component in components:
+        words = component.name.split("-")
+        new_title = ""
+        if words[0] == "fabrikate":
```

Comment:
You can also use a regex: name = re.sub('^fabrikate-', '', name)

Reply:
Nice, this reduced the size of the function significantly!